zh.js

```diff
@@ -547,15 +547,15 @@ windows.controls.taunt2 嘲讽（放屁声）
 windows.controls.taunt3 嘲讽（模因声）
 windows.controls.taunt4 嘲讽（这到底是什么鬼？）
 windows.controls.taunt5 嘲讽（就是现在！）
-windows.controls.prop 选择物体（长按）
-windows.controls.propRand 选择任意物体
-windows.controls.propRot 切换物体旋转角度
-windows.controls.propRotR 重置物体旋转角度
+windows.controls.prop 选择物品（长按）
+windows.controls.propRand 选择任意物品
+windows.controls.propRot 物品旋转切换键
+windows.controls.propRotR 重置物品旋转
 windows.controls.premium0 飞天穿墙
 windows.controls.premium1 无敌
 windows.controls.premium2 自杀
 windows.controls.premium3 瞬间击杀
-windows.controls.reset 重置进度并回到起点（仅限跑酷和速通模式）
+windows.controls.reset 重置进度并回到起点（跑酷等）
```

Comment:
Here I decided to follow en.js's terms (Parkour etc) instead of adding own investigations